pkg/models/jetson-nano-b-sdcard.go

```diff
@@ -0,0 +1,102 @@
+package models
+
+import (
+	"fmt"
+
+	"github.com/lf-edge/eden/pkg/defaults"
+	"github.com/lf-edge/eve/api/go/config"
+	log "github.com/sirupsen/logrus"
+)
+
+//devModelTypeRaspberry is model type for rpi
+const devModelTypeJetsonNanoB devModelType = defaults.DefaultJetsonNanoBModel
+
+//DevModelJetson is dev model fields
+type DevModelJetson struct {
+	//physicalIOs is PhysicalIO slice for DevModel
+	physicalIOs []*config.PhysicalIO
+	//networks is NetworkConfig slice for DevModel
+	networks []*config.NetworkConfig
+	//adapters is SystemAdapter slice for DevModel
+	adapters []*config.SystemAdapter
+}
+
+//Config returns map with config overwrites
+func (ctx *DevModelJetson) Config() map[string]interface{} {
+	cfg := make(map[string]interface{})
+	cfg["eve.serial"] = "*"
+	cfg["eve.remote"] = true
+	cfg["eve.remote-addr"] = ""
+	cfg["eve.arch"] = "arm64"
+	cfg["eve.hostfwd"] = map[string]string{}
+	cfg["eve.devmodel"] = ctx.DevModelType()
+	return cfg
+}
+
+//DiskReadyMessage to show when image is ready
+func (ctx *DevModelJetson) DiskReadyMessage() string {
+	return "Write file %s to sd (it is in raw format)"
+}
+
+//DiskFormat to use for build image
+func (ctx *DevModelJetson) DiskFormat() string {
+	return "raw"
+}
+
+//GetPortConfig returns PortConfig overwrite
+func (ctx *DevModelJetson) GetPortConfig(ssid string, psk string) string {
+	return fmt.Sprintf(`{
```

Comment:
I think, we can return empty string now here. This place is for WiFi.

Reply:
Well, thanks, I will, I didn't notice right away.